pkg/codegen/python/gen_program.go

```diff
@@ -350,7 +350,7 @@ func GenerateProject(
 	if path, ok := localDependencies["pulumi"]; ok {
 		requirementsTxtLines = append(requirementsTxtLines, path)
 	} else {
-		requirementsTxtLines = append(requirementsTxtLines, "pulumi>=3.0.0,<4.0.0")
+		requirementsTxtLines = append(requirementsTxtLines, "pulumi>=3.0.0,<3.153.1")
```

Comment:
This could potentially break any downstream snapshot tests of generated programs expecting `pulumi>=3.0.0,<4.0.0`. Perhaps nobody else is doing that? Still, it'd be unfortunate to cause unnecessary downstream churn for a workaround. I'm wondering if there's another way we could workaround the problem temporarily without having to make this change. Or would we quickly follow-up with a patch release that changes it back to `<4.0.0`?

Reply:
Like temporarily disabling some tests in pu/pu to get the release out? And then re-enable the tests after the release?

Reply:
Yeah, I don't think this is going to be successful anyway.  I'm gonna drop my commit from this PR again.